benchmarks/index.html

```diff
@@ -262,12 +273,22 @@ <h1>Benchmarks</h1>
                     afterTitle: items => {
                       const label = items[0].dataset.label;
                       const index = items[0].dataIndex;
-                      const data = dataset.get(label)[index];
+                      const commitId = commitIds[index];
+                      const dataArr = dataset.get(label);
+                      if (!dataArr) return '';
+                      const data = dataArr.find(d => d.commit.id === commitId);
+                      if (!data) return '';
```

Comment:
Why do we need these lines?

Reply:
Initially, without these lines (and the ones below), the `Lanczos` series datapoint was showing correctly in terms of position along the axis, but the tooltip wasn't showing up. The reason is that it was not correctly matching the commit ID, because, say, index 1 in the `Lanczos` series has not the same commit as index 1 of the `PseudoInverse` series. So we take the correct commit ID from commitIds, and we look for the correct benchmark entry by matching its commit ID.

Reply:
Although I'll admit that rather than opening the correct commit when you click on a datapoint, it opens the first one in the same series. It was doing that even before these changes, though I didn't fix that.